.gitattributes

```diff
@@ -1 +1,3 @@
-**/*.gen.ts linguist-generated=true
\ No newline at end of file
+**/*.gen.ts linguist-generated=true
+**/metadata.ts linguist-generated=true
+**/openAPI.json linguist-generated=true
```

Comment:
Would it make sense to add these to `.prettierignore` so they don't show up as modified while developing? Another option would be to add prettier to the dev watch process, but that seems harder and I'm not certain of the value

Reply:
Done.